jib-gradle-plugin/src/test/resources/gradle/projects/lazy-evaluation/src/main/updated-custom-extra-dir/updated

```diff
@@ -0,0 +1 @@
+updated
```

Comment:
This content is not used for anything? If so, let's have it say "unused" instead of "updated", so people don't search for meaning when looking into this test.

Reply:
Yeah, this file was added to commit its parent directory, so that it is an existing directory for the test project used by `JibPluginTest`. The file name and contents are trivial, I will change this to something less confusing.